aion/config/local_test.exs

```diff
@@ -14,6 +14,6 @@ config :aion, Aion.Repo,
   adapter: Ecto.Adapters.Postgres,
   username: "postgres",
   password: "postgres",
-  database: "aion",
+  database: "aion_test",
```

Comment:
same here ;)

Reply:
This is a one line change that wasn't performed in a previous fix. Sure, I can create another PR for that if you want

Reply:
I don't think it really is needed though. Just like we shouldn't need to create a separate PR for bumping package versions etc. And I think it actually is in the reasonable scope - as long as you can't run tests on this branch with `make test` it's a valid fix imho. 
@pmrukot what is your opinion?

Reply:
I didn't say it should be separate PR :) In my opinion you can easily move it without even making a PR :)

Reply:
Yeah I agree it's kind of a 'no-verify' change and it could be pushed safely without asking others, but I just didn't want to multiply a lot of commits on our master branch. 

Actually, it's a good thing to discuss, we should decide what to do in cases like this in the future. @pmrukot what's your opinion? 

There are advantages and disadvantages to both approaches. Of course the best solution is to push changes that don't require any fixes in the future :joy:

Reply:
Im okay with changing it here. Especially that we overlooked this in few places and I've also changed that in my pr that converted subjects to categories. About introducing changes without a PR, even really minor ones like this - Im against doing that. Easier to track history by PRs than commits on master. But thats my opinion.